src/components/widgets/support-case-widget.tsx

```diff
@@ -0,0 +1,172 @@
+import { Button } from '@patternfly/react-core/dist/dynamic/components/Button';
+import { EmptyState } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateBody } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateIcon } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateVariant } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { Stack } from '@patternfly/react-core/dist/dynamic/layouts/Stack';
+import { StackItem } from '@patternfly/react-core/dist/dynamic/layouts/Stack';
+import { Title } from '@patternfly/react-core/dist/dynamic/components/Title';
+import ExternalLinkAltIcon from '@patternfly/react-icons/dist/dynamic/icons/external-link-alt-icon';
+import { Label } from '@patternfly/react-core/dist/dynamic/components/Label';
+import HeadsetIcon from '@patternfly/react-icons/dist/dynamic/icons/headset-icon';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+import SkeletonTable from '@patternfly/react-component-groups/dist/dynamic/SkeletonTable';
+
+export type Case = {
+  id: string;
+  caseNumber: string;
+  summary: string;
+  lastModifiedById: string;
+  severity: string;
+  status: string;
+};
+
+const SupportCaseWidget: React.FunctionComponent = () => {
+  const [cases, setCases] = useState<Case[]>([]);
+  const MAX_ROWS = 5;
+  const chrome = useChrome();
+  const [isLoading, setIsLoading] = useState(false);
+
+  const columnNames = {
+    caseId: 'Case ID',
+    issueSummary: 'Issue summary',
+    modifiedBy: 'Modified by',
+    severity: 'Severity',
+    status: 'Status',
+  };
+
+  const fetchSupportCases = async () => {
+    const token = await chrome.auth.getToken();
+    const user = await chrome.auth.getUser();
+    const url = 'https://api.access.redhat.com/support/v1/cases/filter';
+    const options = {
+      method: 'POST',
+      headers: {
+        Authorization: `Bearer ${token}`,
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+      },
+      body: JSON.stringify({
+        createdBySSOName: `${user?.identity.user?.username}`,
+      }),
+    };
+
+    try {
+      const response = await fetch(url, options);
+      const { cases } = await response.json();
+      setCases(cases);
+      setIsLoading(false);
+    } catch (error) {
+      console.error('Unable to fetch support cases', error);
+    }
+  };
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetchSupportCases();
+  }, []);
+
+  const labelColor = (severity: string) => {
+    switch (severity) {
+      case '1 (Urgent)':
+        return <Label color="red">{severity}</Label>;
+      case '2 (High)':
+        return <Label color="orange">{severity}</Label>;
+      case '3 (Normal)':
+        return <Label color="blue">{severity}</Label>;
+      case '4 (Low)':
+        return <Label color="grey">{severity}</Label>;
+      default:
+    }
```

Comment:
I am not a big fan of `switch` statement. How about we make an object out of this?
```JS
const labelColor = (severity: string) => {
  const severityMapper = {
    '1 (Urgent)': <Label color="red">{severity}</Label>,
    '2 (High)': <Label color="orange">{severity}</Label>,
    '3 (Normal)': <Label color="blue">{severity}</Label>,
    '4 (Low)': <Label color="grey">{severity}</Label>
  }[severity]
}
```

I would also like to see the severity types to be consts so we can change them whenever we feel like it.